label.rb

```diff
@@ -0,0 +1,20 @@
+require_relative './item'
+require 'date'
+
+class Label < Item
+  attr_accessor :title, :color, :items
+  attr_reader :id, :publish_date
+
+  def initialize(title, color)
+    super(id = Random.rand(1..1000))
```

Comment:
**[Optional]** Since the` Label` class is not an inheritance of the `Item` class, then it should not share the and `@id` with it under `super`. This means there might not be any need for the `super` at all.